polus/plugins.py

```diff
@@ -720,6 +724,123 @@ def update_nist_plugins(gh_auth: typing.Optional[str] = None):
             _error_log(val_err, manifest)
 
 
+class Registry:
+    """Class that contains methods to interact with the REST API of WIPP Registry."""
+
+    def get_current_schema(
+        self,
+        registry_url="https://wipp-registry.ci.aws.labshare.org/",
```

Comment:
Remove the default registry value. Maybe add an `__init__` method and have `registry_url` as an input so that it isn't required in the other methods.